src/multicloud-connector/HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
+Release History
+===============
+
+1.0.0b1
+++++++
+* Initial release.
```

Comment:
`az arc-multicloud public-cloud-connector update` command,

1. Exception rather than an error message. It would be hard for customers to understand stdout exception message, (ObjectMissingRequiredProperty) Missing required property: awsCloudProfile. Paths in payload: '$.properties.awsCloudProfile'
 
![image](https://github.com/user-attachments/assets/00967996-de9d-4ed6-ae97-8d3ebd155a16)

2. connector is created as per the example, but the response is wrong as the awsCloudProfile is empty. This is also a bug on our controller. Please file a bug to fix the controller.
**Help message**
![image](https://github.com/user-attachments/assets/3d583826-a6fa-43aa-8817-69c12e8e5fbb)

**Connector Response**
![image](https://github.com/user-attachments/assets/1b9135a0-f2be-4baf-8cd5-e208f4d7cdba)

3. As an end user, it's very hard to understand what should go into the `aws-cloud-profile`. The help should be very clear and should also point to our public documentation page that capture various options. 
![image](https://github.com/user-attachments/assets/cec112dd-41ac-4d8f-b680-7f2e053dcc81)